provisioner/terraform/executor.go

```diff
@@ -359,9 +360,15 @@ func (e *executor) planResources(ctx, killCtx context.Context, planfilePath stri
 
 	state, err := ConvertState(ctx, modules, rawGraph, e.server.logger)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
-	return state, nil
+
+	planJSON, err := json.Marshal(plan)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return state, planJSON, nil
```

Comment:
Unfortunate we have to `json.Marshal` when we had to decode the original json. But this could be seen as a hygienic way to ensure a clean json :thinking: 

This is fine :+1: